sdk/src/types/block/output/mod.rs

```diff
@@ -416,3 +433,15 @@ pub trait MinimumOutputAmount: StorageScore {
         self.storage_score(params) * params.storage_cost()
     }
 }
+
+/// Decayed stored and potential Mana of an output.
+#[derive(Debug, Clone, Eq, PartialEq, Ord, PartialOrd, Hash)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct DecayedMana {
+    /// Decayed stored mana.
+    #[cfg_attr(feature = "serde", serde(with = "string"))]
+    pub stored: u64,
+    /// Decayed potential mana.
+    #[cfg_attr(feature = "serde", serde(with = "string"))]
+    pub potential: u64,
```

Comment:
I'd remove the pubs and put getters, shouldn't be able to edit these without the formulas